### ISSUE TITLE
How to use it with Remote SSH?

### ISSUE THREAD

**Issue by vonsluzbov** [CSeitel/open-in-npp#5]
I have VSC GUI running under Windows, but working via Remote SSH. When I tried to use th extention (either via Ctrl+K Ctrl+N, or context menu) I got:

Notepad++ execution failed due to error: "spawn C:\Program Files\Notepad++\notepad++.exe ENOENT"

I tried to setup executable path both for remote and local - it didn't help.

**Comment by CSeitel** [comment]
Are you referring to a scenario as described here [https://code.visualstudio.com/docs/remote/ssh](https://code.visualstudio.com/docs/remote/ssh)? The error would mean: File "_C:\Program Files\Notepad++\notepad++.exe_" not found likely from the perspective of the remote server.

**Comment by vonsluzbov** [comment]
Yes, exactly, this is the scenario.  Shoudn't the vscode look for the notepad in a given directory ?

**Comment by CSeitel** [comment]
The extension runs on the remote server and starting _Notepad++_ there is likely not of much use because you would want to start it on your _VSC GUI Windows_ system  with accessing the remote files in a way as described here [NppFTP - SSH](https://www.idc-online.com/technical_references/pdfs/information_technology/How_to_edit_files_using_Notepad_DoublePlus_over_SSH_File_Transfer_Protocol.pdf).

The path to the _Notepad++_ -executable can be set in the settings of the extension as potentially e.g. _C:\Program Files **(x86)**_ was also a viable location in the past. In the remote _SSH_-scenario I am not sure whether the root cause of the error _ENOENT = FileNotFound_ couldn't also come from missing file access permissions.
